es/es-server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -69,10 +69,12 @@
 
     public static final int ES_V_7_0_0_ID = 7_00_00_99;
     public static final int ES_V_7_0_1_ID = 7_00_01_99;
-    public static final Version V_4_0_0 = new Version(ES_V_7_0_0_ID, ES_V_7_0_0_ID - INTERNAL_OFFSET, org.apache.lucene.util.Version.LUCENE_8_0_0);
-    public static final Version V_4_0_1 = new Version(ES_V_7_0_1_ID, ES_V_7_0_1_ID - INTERNAL_OFFSET, true, org.apache.lucene.util.Version.LUCENE_8_0_0);
+    public static final int ES_V_7_1_0_ID = 7_01_00_99;
+    public static final Version V_4_0_0 = new Version(ES_V_7_0_0_ID, false, org.apache.lucene.util.Version.LUCENE_8_0_0);
+    public static final Version V_4_0_1 = new Version(ES_V_7_0_1_ID, false, org.apache.lucene.util.Version.LUCENE_8_0_0);
```

Comment:
4.0.1 is still snapshot?

Reply:
Somehow yes, but I'm not sure if we want to track it as such here on master. 

Might make sense to remove the unused static declarations altogether and allow `fromId` to create Version instances dynamically.